css/stylesheet.css

```
@@ -153,4 +153,54 @@ p span {
     line-height: 20px;
     font-weight: bold;
     transform: translate( 0, -120px);
-}
\ No newline at end of file
+}
+
+.introduce {
+    margin: 100px auto;
+    width: 90%; 
+}
```

MUST
今回の場合は、widthは必要ないと思いました。
セクションの余白はpaddingでつける方がいいと思います。
flexで並べている会社のロゴは、justify-content: center;で既に中央寄せされているからです。

参考サイトを見ると、PC幅を縮めたときに最小4カラムになっていたので、
このままだとカラム落ちしてしまうと思います。
（レスポンシブは別issueなので、paddingの値の調整は後で良いかもしれませんが、leftとright 20pxくらいでしょうか・・！）
```suggestion
.introduce {
    padding: 128px 20px; // セクションのtop,bottomも私はpaddingで入れます。
}
```

---

違いが気になったので今変更しました。
paddingでいいのですね…すごい…！！
ありがとうございます！

3697593

---

すみません、.company-list にmax-widthが必要でした🙇‍♀️
画面を広げていくと、カラム数が６個以上に増えていってしまいました。

```
.company-list {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  max-width: 1280px; //追加
  margin: 40px auto 0 auto; //追加
}
```
こちらでリストの最大幅が決まっているので、
introduceセクションの左右のpaddingはあってもなくても良いです🙇‍♀️
確認不足で申し訳ありません・・・！

---

ありがとうございます！
確認して、修正しました。

a50605d